packages/main/src/translations/translation-dictionary/components/Preferences.js

```diff
@@ -128,14 +128,8 @@ export default [
   {
     key: "language",
     en: "Language",
-
     fr: "Langue",
-    zh: "语言",
-    ar: "لغة",
-    es: "Idioma",
-    it: "Lingua",
-    iw: "שפה",
-    pt: "Idioma"
+    de: "Taal"
```

Comment:
Arent the languages you removed above being used?

Reply:
They are. I will add them right now